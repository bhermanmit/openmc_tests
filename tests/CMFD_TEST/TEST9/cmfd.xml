<?xml version="1.0"?>
<cmfd>

  <mesh>
    <origin> -15 -1 -1 </origin>
    <dimension> 30 1 1 </dimension>
    <width> 1.0 2.0 2.0 </width>
    <albedo> 0.0 0.0 1.0 1.0 1.0 1.0 </albedo>
    <map> 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 </map>
    <energy> 0.0 1.0 20.0 </energy>
  </mesh>

</cmfd>
