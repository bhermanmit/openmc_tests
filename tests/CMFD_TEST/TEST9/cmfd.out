MESH (1. 1, 1)  GROUP 1           3.30000E-03
MESH (1. 1, 1)  GROUP 2           -3.30000E-03
MESH (2. 1, 1)  GROUP 1           3.78000E-04
MESH (2. 1, 1)  GROUP 2           -3.78000E-04
MESH (3. 1, 1)  GROUP 1           -1.41000E-04
MESH (3. 1, 1)  GROUP 2           1.41000E-04
MESH (4. 1, 1)  GROUP 1           -1.78000E-04
MESH (4. 1, 1)  GROUP 2           1.78000E-04
MESH (5. 1, 1)  GROUP 1           3.31000E-03
MESH (5. 1, 1)  GROUP 2           -3.31000E-03
MESH (6. 1, 1)  GROUP 1           -2.05841E-03
MESH (6. 1, 1)  GROUP 2           2.06892E-03
MESH (7. 1, 1)  GROUP 1           1.34152E-04
MESH (7. 1, 1)  GROUP 2           -1.07821E-04
MESH (8. 1, 1)  GROUP 1           8.20367E-04
MESH (8. 1, 1)  GROUP 2           -7.96761E-04
MESH (9. 1, 1)  GROUP 1           7.80984E-04
MESH (9. 1, 1)  GROUP 2           -8.10658E-04
MESH (10. 1, 1)  GROUP 1          9.80517E-04
MESH (10. 1, 1)  GROUP 2          -1.03242E-03
MESH (11. 1, 1)  GROUP 1          1.17276E-03
MESH (11. 1, 1)  GROUP 2          -1.18109E-03
MESH (12. 1, 1)  GROUP 1          1.07304E-03
MESH (12. 1, 1)  GROUP 2          -1.07782E-03
MESH (13. 1, 1)  GROUP 1          9.36387E-04
MESH (13. 1, 1)  GROUP 2          -9.48025E-04
MESH (14. 1, 1)  GROUP 1          1.21867E-03
MESH (14. 1, 1)  GROUP 2          -1.13763E-03
MESH (15. 1, 1)  GROUP 1          1.13136E-03
MESH (15. 1, 1)  GROUP 2          -1.04913E-03
MESH (16. 1, 1)  GROUP 1          1.08092E-03
MESH (16. 1, 1)  GROUP 2          -1.09993E-03
MESH (17. 1, 1)  GROUP 1          1.00977E-03
MESH (17. 1, 1)  GROUP 2          -9.95027E-04
MESH (18. 1, 1)  GROUP 1          1.10652E-03
MESH (18. 1, 1)  GROUP 2          -1.09388E-03
MESH (19. 1, 1)  GROUP 1          1.14552E-03
MESH (19. 1, 1)  GROUP 2          -1.28616E-03
MESH (20. 1, 1)  GROUP 1          8.62488E-04
MESH (20. 1, 1)  GROUP 2          -8.65922E-04
MESH (21. 1, 1)  GROUP 1          1.12035E-03
MESH (21. 1, 1)  GROUP 2          -1.15673E-03
MESH (22. 1, 1)  GROUP 1          1.09807E-03
MESH (22. 1, 1)  GROUP 2          -1.15359E-03
MESH (23. 1, 1)  GROUP 1          8.32871E-04
MESH (23. 1, 1)  GROUP 2          -7.20512E-04
MESH (24. 1, 1)  GROUP 1          3.37226E-04
MESH (24. 1, 1)  GROUP 2          -3.48414E-04
MESH (25. 1, 1)  GROUP 1          -2.24813E-03
MESH (25. 1, 1)  GROUP 2          2.25716E-03
MESH (26. 1, 1)  GROUP 1          3.17200E-03
MESH (26. 1, 1)  GROUP 2          -3.17200E-03
MESH (27. 1, 1)  GROUP 1          1.32000E-04
MESH (27. 1, 1)  GROUP 2          -1.32000E-04
MESH (28. 1, 1)  GROUP 1          -2.62000E-04
MESH (28. 1, 1)  GROUP 2          2.62000E-04
MESH (29. 1, 1)  GROUP 1          3.51000E-04
MESH (29. 1, 1)  GROUP 2          -3.51000E-04
MESH (30. 1, 1)  GROUP 1          3.38600E-03
MESH (30. 1, 1)  GROUP 2          -3.38600E-03
